### ISSUE TITLE
Failing test System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatforms

### ISSUE THREAD

**Issue by radical** [dotnet/runtime#87038]
## Build Information
Build: https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_build/results?buildId=293100
Build error leg or test failing: System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatforms_IsSupportedIsTrue
Pull request: https://github.com/dotnet/runtime/pull/86048

## Error Message

Fill the error message using [step by step known issues guidance](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssueJsonStepByStep.md#how-to-create-a-known-issue-step-by-step).

```json
{
  "ErrorMessage": "System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatforms",
  "ErrorPattern": "",
  "BuildRetry": false,
  "ExcludeConsoleLog": false
}
```

### Report

|Build|Definition|Test|Pull Request|
|---|---|---|---|
|[293351](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293351)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293351&view=ms.vss-test-web.build-test-results-tab&runId=5891034&resultId=183141)|dotnet/runtime#87044|
|[293346](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293346)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293346&view=ms.vss-test-web.build-test-results-tab&runId=5890992&resultId=183052)|dotnet/runtime#84191|
|[293332](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293332)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293332&view=ms.vss-test-web.build-test-results-tab&runId=5890502&resultId=176391)|dotnet/runtime#86892|
|[293320](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293320)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293320&view=ms.vss-test-web.build-test-results-tab&runId=5890228&resultId=178083)|dotnet/runtime#87042|
|[293313](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293313)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293313&view=ms.vss-test-web.build-test-results-tab&runId=5889936&resultId=175547)|dotnet/runtime#85392|
|[293309](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293309)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293309&view=ms.vss-test-web.build-test-results-tab&runId=5889866&resultId=176681)|dotnet/runtime#86954|
|[293273](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293273)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293273&view=ms.vss-test-web.build-test-results-tab&runId=5888514&resultId=176002)|dotnet/runtime#87002|
|[293271](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293271)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293271&view=ms.vss-test-web.build-test-results-tab&runId=5888452&resultId=158703)|dotnet/runtime#85373|
|[293231](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293231)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293231&view=ms.vss-test-web.build-test-results-tab&runId=5887540&resultId=176743)|dotnet/runtime#86813|
|[293225](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293225)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293225&view=ms.vss-test-web.build-test-results-tab&runId=5887386&resultId=177249)|dotnet/runtime#86892|
|[293204](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293204)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293204&view=ms.vss-test-web.build-test-results-tab&runId=5886574&resultId=177224)|dotnet/runtime#86974|
|[293199](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293199)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293199&view=ms.vss-test-web.build-test-results-tab&runId=5886158&resultId=182509)|dotnet/runtime#85558|
|[293106](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293106)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293106&view=ms.vss-test-web.build-test-results-tab&runId=5884206&resultId=178228)|dotnet/runtime#85373|
|[293100](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293100)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=293100&view=ms.vss-test-web.build-test-results-tab&runId=5883920&resultId=176956)|dotnet/runtime#86048|
|[292967](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292967)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292967&view=ms.vss-test-web.build-test-results-tab&runId=5880728&resultId=177577)|dotnet/runtime#87025|
|[292955](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292955)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292955&view=ms.vss-test-web.build-test-results-tab&runId=5880700&resultId=175765)||
|[292927](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292927)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292927&view=ms.vss-test-web.build-test-results-tab&runId=5880326&resultId=166040)|dotnet/runtime#86578|
|[292880](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292880)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292880&view=ms.vss-test-web.build-test-results-tab&runId=5879230&resultId=174168)|dotnet/runtime#86892|
|[292824](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292824)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292824&view=ms.vss-test-web.build-test-results-tab&runId=5878294&resultId=172849)|dotnet/runtime#86983|
|[292798](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292798)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292798&view=ms.vss-test-web.build-test-results-tab&runId=5876960&resultId=176809)|dotnet/runtime#86910|
|[292786](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292786)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292786&view=ms.vss-test-web.build-test-results-tab&runId=5876848&resultId=171953)|dotnet/runtime#86806|
|[292787](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292787)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292787&view=ms.vss-test-web.build-test-results-tab&runId=5876760&resultId=169936)|dotnet/runtime#86806|
|[292710](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292710)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292710&view=ms.vss-test-web.build-test-results-tab&runId=5875372&resultId=174749)|dotnet/runtime#86813|
|[292503](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292503)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292503&view=ms.vss-test-web.build-test-results-tab&runId=5875198&resultId=176250)|dotnet/runtime#86913|
|[292701](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292701)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292701&view=ms.vss-test-web.build-test-results-tab&runId=5875118&resultId=172196)|dotnet/runtime#87019|
|[292653](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292653)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292653&view=ms.vss-test-web.build-test-results-tab&runId=5873668&resultId=173496)|dotnet/runtime#86974|
|[292587](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292587)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292587&view=ms.vss-test-web.build-test-results-tab&runId=5870902&resultId=177302)|dotnet/runtime#86787|
|[292582](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292582)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292582&view=ms.vss-test-web.build-test-results-tab&runId=5870706&resultId=176150)|dotnet/runtime#86578|
|[292552](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292552)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292552&view=ms.vss-test-web.build-test-results-tab&runId=5870362&resultId=175304)|dotnet/runtime#86998|
|[292543](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292543)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292543&view=ms.vss-test-web.build-test-results-tab&runId=5870156&resultId=177225)|dotnet/runtime#86975|
|[292478](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292478)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292478&view=ms.vss-test-web.build-test-results-tab&runId=5867240&resultId=175338)|dotnet/runtime#86035|
|[292460](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292460)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292460&view=ms.vss-test-web.build-test-results-tab&runId=5866656&resultId=176869)|dotnet/runtime#87007|
|[292441](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292441)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292441&view=ms.vss-test-web.build-test-results-tab&runId=5866532&resultId=180950)|dotnet/runtime#87006|
|[292358](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292358)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292358&view=ms.vss-test-web.build-test-results-tab&runId=5864120&resultId=174448)|dotnet/runtime#86255|
|[292333](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292333)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292333&view=ms.vss-test-web.build-test-results-tab&runId=5863552&resultId=183225)|dotnet/runtime#87003|
|[292307](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292307)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292307&view=ms.vss-test-web.build-test-results-tab&runId=5861982&resultId=172994)|dotnet/runtime#87002|
|[292295](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292295)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292295&view=ms.vss-test-web.build-test-results-tab&runId=5861710&resultId=169005)|dotnet/runtime#87001|
|[292251](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292251)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292251&view=ms.vss-test-web.build-test-results-tab&runId=5860154&resultId=178222)|dotnet/runtime#86652|
|[292245](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292245)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292245&view=ms.vss-test-web.build-test-results-tab&runId=5860010&resultId=180280)|dotnet/runtime#87000|
|[292240](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292240)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292240&view=ms.vss-test-web.build-test-results-tab&runId=5859802&resultId=170318)|dotnet/runtime#86698|
|[292199](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292199)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292199&view=ms.vss-test-web.build-test-results-tab&runId=5857860&resultId=173547)|dotnet/runtime#86578|
|[291378](https://dev.azure.com/dnceng-public/public/_build/results?buildId=291378)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=291378&view=ms.vss-test-web.build-test-results-tab&runId=5857788&resultId=177729)|dotnet/runtime#86766|
|[292191](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292191)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292191&view=ms.vss-test-web.build-test-results-tab&runId=5857776&resultId=175712)|dotnet/runtime#86787|
|[292163](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292163)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292163&view=ms.vss-test-web.build-test-results-tab&runId=5857092&resultId=176554)|dotnet/runtime#85378|
|[292114](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292114)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292114&view=ms.vss-test-web.build-test-results-tab&runId=5855768&resultId=173633)|dotnet/runtime#86995|
|[292110](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292110)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292110&view=ms.vss-test-web.build-test-results-tab&runId=5855418&resultId=175583)|dotnet/runtime#86994|
|[292068](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292068)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292068&view=ms.vss-test-web.build-test-results-tab&runId=5854398&resultId=171873)|dotnet/runtime#86470|
|[292063](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292063)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292063&view=ms.vss-test-web.build-test-results-tab&runId=5854278&resultId=170704)|dotnet/runtime#86928|
|[292041](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292041)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292041&view=ms.vss-test-web.build-test-results-tab&runId=5853796&resultId=173628)|dotnet/runtime#85392|
|[292035](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292035)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=292035&view=ms.vss-test-web.build-test-results-tab&runId=5853724&resultId=173624)||
|[291986](https://dev.azure.com/dnceng-public/public/_build/results?buildId=291986)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=291986&view=ms.vss-test-web.build-test-results-tab&runId=5852088&resultId=175565)|dotnet/runtime#86813|
|[291972](https://dev.azure.com/dnceng-public/public/_build/results?buildId=291972)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=291972&view=ms.vss-test-web.build-test-results-tab&runId=5851866&resultId=163392)|dotnet/runtime#84191|
|[291957](https://dev.azure.com/dnceng-public/public/_build/results?buildId=291957)|dotnet/runtime|[System.Net.Quic.Tests.MsQuicPlatformDetectionTests.SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue](https://dev.azure.com/dnceng-public/public/_build/results?buildId=291957&view=ms.vss-test-web.build-test-results-tab&runId=5851218&resultId=168877)|dotnet/runtime#85558|
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|48|53|53|

**Comment by ViktorHofer** [comment]
cc @dotnet/ncl

**Comment by wfurt** [comment]
```
[helixbot@cc5f0702390f /]$ ldd /usr/lib64/libmsquic.so.2.2.1
	linux-vdso.so.1 (0x00007ffd43b3e000)
	libcrypto.so.1.1 => /lib64/libcrypto.so.1.1 (0x00007feb1cc1f000)
	libdl.so.2 => /lib64/libdl.so.2 (0x00007feb1ca1b000)
	libnuma.so.1 => not found
	libpthread.so.0 => /lib64/libpthread.so.0 (0x00007feb1c7fb000)
	libc.so.6 => /lib64/libc.so.6 (0x00007feb1c436000)
	libz.so.1 => /lib64/libz.so.1 (0x00007feb1c21e000)
	/lib64/ld-linux-x86-64.so.2 (0x00007feb1d109000)
```

this goes back to https://github.com/microsoft/msquic/issues/3421 in MsQuic. 

It seems like this was pulled in by https://github.com/dotnet/runtime/pull/86342 @ManickaP

**Comment by ManickaP** [comment]
Could someone explain to me how this started failing in runtime pipeline if that pipeline was passing in my PR https://github.com/dotnet/runtime/pull/86342?
The same test on the same platform was passing in my PR, what has changed???

**Comment by wfurt** [comment]
MsQuic? 
```
Version     : 2.2.1
Build Date  : Wed 24 May 2023 12:28:49 PM UTC 
```

I'm wondering if https://github.com/microsoft/msquic/issues/3619 got propagated somehow @nibanks

**Comment by CarnaViire** [comment]
Is it possible that packages got republished for all platforms in https://github.com/microsoft/msquic/pull/3652 ? @nibanks @csujedihy

**Comment by nibanks** [comment]
v2.1.1 had the fix for actually consuming libnuma (where v2.2.0 didn't), but I don't think that changed since the package was published. @csujedihy to confirm

**Comment by csujedihy** [comment]
v2.2.1 doesn't have the libnuma dep fix but https://github.com/microsoft/msquic/pull/3652 uploaded packages versioned as v2.2.1 but actually have newer slightly payload, which includes the libnum dep fix.

The new PMC v4 API seems to have override the old v2.2.1 packages.

@CarnaViire @wfurt Is there anything we should do to help fix this?
